src/_designs/components/timeStamp/index.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div
+    class="time-stamp"
+    :class="{ header: hasTitle }"
+  >
+    <AeText
```

Comment:
in this case we dont need the AeText component as you also set the same font styles. I would opt for a span instead

Reply:
done